avocado/utils/linux.py

```diff
@@ -51,3 +53,12 @@ def set_proc_sys(key, value):
     path = os.path.join('/proc/sys/%s', key)
     genio.write_one_line(path, value)
     return get_proc_sys(key)
+
+
+def selinux_enforcing():
```

Comment:
Small suggestion here: this function name suggests that this could enable selinux. So, maybe something like: `is_selinux_enforcing()`. What do you think?

Reply:
ah sure make sense will adopt this